app/controllers/report_controller/reports/editor.rb

```diff
@@ -1164,7 +1164,12 @@ def tag_values_from(tag_or_tags)
   end
 
   def parse_tag_categories(category, tag_values)
-    tag_values.split(",").map { |tag_value| "/managed/#{category}/#{tag_value}" }
+    case tag_values 
+    when String
+      tag_values.split(",").map { |tag_value| "/managed/#{category}/#{tag_value}" }  
+    when Array
+      tag_values.map { |tag_value| "/managed/#{category}/#{tag_value}" }
+    end
```

Comment:
Slightly cleaner to avoid duplicating the map

```suggestion
    tag_values = tag_values.split(",") if tag_values.kind_of?(string)
    tag_values.map { |tag_value| "/managed/#{category}/#{tag_value}" }
```